### ISSUE TITLE
Failed to compile error on build

### ISSUE THREAD

**Issue by TxHawks** [styleguidist/react-styleguidist#782]
When using the build command, I get a 
```sh
Failed to compile.
build/bundle.9ad148f6.js from UglifyJs
undefined
```
This is obviously not an issue with styleguidist, but rather an UglifyJs issue, and everything works fine when I disable minification in Webpack, but the error message is not very helpful.

If anyone has any idea on how to get a more productive error message, I'd be very greatful

**Comment by sapegin** [comment]
I guess we can do something with this `undefined`. Please make a demo project that we can debug based on this repo: https://github.com/styleguidist/example

**Comment by TxHawks** [comment]
Problem is, I'm not really sure what's causing this, so hard to reproduce, but I'll try.

**Comment by sapegin** [comment]
Most likely you can find more information about the error by adding some `console.log`s here:

https://github.com/styleguidist/react-styleguidist/blob/7ab1ebf4245896ccad42c0a133987dff8869c72f/bin/styleguidist.js#L89-L112

**Comment by TxHawks** [comment]
Fixed by #797
